nx-dev/ui-customers/src/lib/customer-testimonial-carousel.tsx

```diff
@@ -0,0 +1,400 @@
+    metrics: [
+      {
+        value: '2x faster',
+        label: 'Nx doubles speed, supporting fast, nimble development.',
+      },
+      {
+        value: 'Unmatched DX',
+        label: 'Nx empowers teams to be more productive than ever.',
+      },
+      {
+        value: 'Always ahead',
+        label:
+          'Nx’s rapid feature delivery keeps teams at the cutting edge of development.',
+      },
+    ],
+    company: 'Broadcom (VMware)',
+    videoId: 'RWTgYNKqxNc',
+    thumbnail: '/images/customers/video-story-broadcom.avif',
+    logo: {
+      icon: VmwareIcon,
+      height: 'h-28',
+      width: 'w-28',
+    },
+  },
+];
+
+export function CustomerTestimonialCarousel(): JSX.Element {
+  const [currentIndex, setCurrentIndex] = useState(0);
+  const [isOpen, setIsOpen] = useState(false);
+  const currentTestimonial = testimonials[currentIndex];
+  const slideLogoTimeOut = 12000; // 12 seconds
+
+  useEffect(() => {
+    let timer: NodeJS.Timeout;
+
+    if (!isOpen) {
+      timer = setInterval(() => {
+        setCurrentIndex((prevIndex) => {
+          const nextIndex = (prevIndex + 1) % testimonials.length;
+          return nextIndex;
+        });
+      }, slideLogoTimeOut);
+    }
+
+    return () => {
+      clearInterval(timer);
+    };
+  }, [currentIndex, setCurrentIndex, isOpen]);
+
+  return (
+    <div className="w-full">
+      <div className="mx-auto grid max-w-7xl grid-cols-2 gap-2 px-4 lg:grid-cols-5 lg:gap-4">
+        {/* Left side - Quote or Metrics */}
+        <div className="col-span-2 hidden lg:block">
+          <div className="flex h-full flex-col justify-center space-y-8">
+            {currentTestimonial.metrics?.map((metric, index) => (
+              <div key={index} className="space-y-2">
+                <div className="border-l-2 border-sky-900/70 pl-4 text-3xl font-bold text-slate-700 dark:border-blue-300/60 dark:text-slate-200">
+                  {metric.value}
+                </div>
+                <div className="text-balance pl-[18px] text-lg text-slate-500 dark:text-slate-400">
+                  {metric.label}
+                </div>
+              </div>
+            ))}
+          </div>
+        </div>
+
+        {/* Right side - Video Card */}
+        <div className="col-span-2 md:col-span-3">
+          <div className="flex items-center gap-4">
+            {/* Prev Button Mobile only */}
+            <button
+              disabled={currentIndex === 0}
+              title={`See ${testimonials[currentIndex - 1]?.company} again!`}
+              className="flex h-12 w-12 items-center justify-center rounded-full p-2 transition hover:text-slate-950 disabled:pointer-events-none disabled:opacity-0 md:hidden dark:hover:text-white"
+              onClick={() => {
+                setCurrentIndex(
+                  (currentIndex - 1 + testimonials.length) % testimonials.length
+                );
+              }}
+            >
+              <ChevronLeftIcon className="h-8 w-8" />
```

Comment:
Add `flex-shrink-0` to make sure the icon is not resized,  `aria-hidden="true"` to make it invisible to screen reader.